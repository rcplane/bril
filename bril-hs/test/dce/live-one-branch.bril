@main {
  # This def is live because i is reassigned in only one path
  i: int = const 0;
  cond: bool = const true;
  br cond .hi .bye;
.hi:
  i: int = const 1;
.bye:
  print i;
}
