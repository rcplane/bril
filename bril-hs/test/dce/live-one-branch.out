@main {
  i: int = const 0;
  cond: bool = const true;
  br cond .hi .bye;
.hi:
  i: int = const 1;
.bye:
  print i;
}
